apps/zotonic_core/src/models/m_edge.erl

```diff
@@ -216,7 +216,7 @@ insert1(SubjectId, PredId, ObjectId, Opts, Context) ->
             F = fun(Ctx) ->
                 case z_convert:to_bool(proplists:get_value(no_touch, Opts, false)) of
                     true -> skip;
-                    false -> m_rsc:touch(SubjectId, Ctx)
+                    false -> pivot_resources([SubjectId, ObjectId], Ctx)
```

Comment:
@mworrell As you can see, `no_touch` now prevents the subject and object from being pivoted. Should we rename this option to `no_pivot`? I Or simply remove it?